besu/src/main/java/org/hyperledger/besu/controller/CliqueBesuControllerBuilder.java

```diff
@@ -84,6 +85,7 @@ protected MiningCoordinator createMiningCoordinator(
       final ProtocolContext protocolContext,
       final TransactionPool transactionPool,
       final MiningConfiguration miningConfiguration,
+      final PeerTaskExecutor peerTaskExecutor,
```

Comment:
Yep, good idea

Reply:
Unfortunately, I can't really find a way around it. Overloading won't help because it's called from within BesuControllerBuilder, which can't make the distinction between which subclasses need the PeerTaskExecutor and which don't